common/dao-api/src/main/java/org/thingsboard/server/dao/attributes/AttributesService.java

```diff
@@ -47,4 +48,6 @@ public interface AttributesService {
 
     List<String> findAllKeysByEntityIds(TenantId tenantId, EntityType entityType, List<EntityId> entityIds);
 
+    List<AttributeKvEntryEntityId> findAllValuesByEntityIds(TenantId tenantId, String attributeKey, List<EntityId> entityIds);
```

Comment:
I believe more correct name for this method is 
**findValuesByKeyAndEntityIds**